app/src/main/kotlin/com/wire/android/feature/conversation/content/ui/ConversationFragment.kt

```diff
@@ -59,6 +59,6 @@ class ConversationFragment : Fragment(R.layout.fragment_conversation) {
 
     override fun onPause() {
         super.onPause()
-        viewModel.updateCurrentConversationId(String.EMPTY)
+        viewModel.updateCurrentConversationId(ConversationID.blankID())
```

Comment:
_Maybe_ when it was a String it made some sense to do this way. But at this point, I guess making the `currentConversationId` nullable is just better.

Using `String.EMPTY` was arguably worse than making it nullable anyway. Kotlin has null safety, and it's awesome that we can use it.

If we don't have a current conversation ID, we can just make it `null`. That's not ugly, that's easy to understand.

If I needed to use the `currentOpenedConversationId` function, and I wanted to know if there was any conversation ID, I would probably check for nullability, left side of an `Either`, `String.EMPTY` or `ConversationID.blankID()` are not discoverable.